WPObjectMapper/NSObject+ObjectMap.m

```diff
@@ -39,9 +39,10 @@
 @protocol NSObjectMappable
 @optional
 + (NSDictionary *) mapping;
-+ (instancetype) objectWithDictionary: (NSDictionary *) dictionary;
++ (nullable instancetype) objectWithDictionary: (NSDictionary *) dictionary;
 @end
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I don't think this is right.  I think the following all must be nullable:

```objc
- (instancetype)initWithJSONData:(NSData *)data{

- (instancetype)initWithXMLData:(NSData *)data{

- (instancetype)initWithSOAPData:(NSData *)data{

- (instancetype)initWithObjectData:(NSData *)data type:(CAPSDataType)type {

+ (NSArray *)arrayOfType:(Class)objectClass FromJSONData:(NSData *)data {

+(id)objectOfClass:(Class)objectClass fromXML:(NSString *)xml {

-(id)getNodeValue:(NSString *)node fromXML:(NSString *)xml {

+ (id)objectOfClass:(Class)objectClass fromJSONData:(NSData *)jsonData {

+ (NSDate *) parseDate:(id) dateValue {

-(NSString *)classOfPropertyNamed:(NSString *)propName {

-(NSString *)nameOfClass {

-(NSString *)typeFromProperty:(objc_property_t)property {

-(NSData *)JSONData{

-(NSString *)JSONString{

-(NSData *)SOAPData{

-(NSData *)XMLData{

+ (NSString *)encodeBase64WithData:(NSData *)objData {

```

I may be missing some, so please double-check me on this.